src/schema/validator.js

```diff
@@ -113,6 +101,34 @@ export const validateStaticTheme = (...args) => {
   return isValid;
 };
 
+// Like with static themes, we don't want additional properties in langpacks.
+// The only difference is, this time, there is no additional schema file, we
+// just need to reference WebExtensionLangpackManifest and merge it with the
+// object that has additionalProperties: false.
+const _validateLangPack = validator.compile({
```

Comment:
That seems to work, but let me know if I messed it up :)

Reply:
Looks good to me and works nicely.

Ideally this should all be upstream but it's a completely different story and time-intensive.